tests/test_textract/test_server.py

```diff
@@ -0,0 +1,13 @@
+"""Test different server responses."""
+import sure  # noqa # pylint: disable=unused-import
+
+import moto.server as server
+
+
+def test_textract_list():
+    backend = server.create_backend_app("textract")
+    test_client = backend.test_client()
+
+    resp = test_client.get("/")
+    resp.status_code.should.equal(200)
+    str(resp.data).should.contain("?")
```

Comment:
This test is automatically generated - the idea behind it is that we send a raw HTTP request to the server, to verify that Moto understands it and responds accordingly.
The other tests verify that Moto  understands requests created by boto3 - this one really acts as a belts and braces.

A simple example on how to create a HTTP request:
https://github.com/spulec/moto/blob/master/tests/test_cloudtrail/test_server.py

As Textract does not have any straightforward `list_...` methods (yet). So the simplest test case would probably be to verify that `get_document_text_detection` throws an error message if no JobId is send through.
The appropriate header for Textract looks like this: `X-Amz-Target=Textract.GetDocumentTextDetection`

Apologies for the lack of documentation here - that is still a work in progress :slightly_smiling_face: